components/script/dom/readablestream.rs

```diff
@@ -414,7 +416,11 @@ impl ReadableStream {
                     .expect("Stream must have a reader when a read request is fulfilled.");
                 let request = reader.remove_read_request();
                 if !done {
-                    request.chunk_steps(chunk);
+                    let cx = GlobalScope::get_cx();
+                    rooted!(in(*cx) let mut rval = UndefinedValue());
+                    let result = RootedTraceableBox::new(Heap::default());
+                    result.set(*chunk);
+                    request.chunk_steps(result);
```

Comment:
this logic kinda duplicated with `perform_pull_steps ` but I think we can merge it like this

Reply:
Yes I propose we do this quickly and revisit when the dust has settled.